config/config.go

```diff
@@ -231,14 +230,6 @@ type BaseConfig struct { //nolint: maligned
 
 // DefaultBaseConfig returns a default base configuration for an Ostracon node
 func DefaultBaseConfig() BaseConfig {
-	dbs := metadb.AvailableDBBackends()
```

Comment:
`AvailableDBBackends` is `line/tm-db` original.
* https://github.com/line/tm-db/pull/15